lbrynet/reflector/client/client.py

```diff
@@ -420,7 +424,7 @@ def send_next_request(self):
             d = self.blob_manager.get_blob(blob_hash, True)
             d.addCallback(self.open_blob_for_reading)
             # send the server the next blob hash + length
-            d.addCallback(lambda _: self.send_blob_info())
+            d.addCallbacks(lambda _: self.send_blob_info(), lambda err: self.log_fail_and_disconnect(err, blob_hash))
```

Comment:
can you put this on two lines.  I have a hard time reading this to see what the two callbacks are?